mis_builder/models/mis_report_instance.py

```diff
@@ -320,9 +320,9 @@ def _compute_pivot_date(self):
                 record.pivot_date = fields.Date.context_today(record)
 
     @api.model
-    def _default_company(self):
-        return self.env['res.company'].\
-            _company_default_get('mis.report.instance')
+    def _default_company_ids(self):
+        return [(6,0,[self.env['res.company'].\
+            _company_default_get('mis.report.instance').id])]
```

Comment:
> I'm wondering if return _company_default_get('mis.report.instance') would work too?

That's what I was using first, but there were a json complaint about res.company not serialisable.
Why flake8 will complain?